3rdparty/find_dependencies.cmake

```diff
@@ -901,14 +922,22 @@ if(BUILD_GUI)
     set(FILAMENT_MATC "${FILAMENT_ROOT}/bin/matc")
     target_link_libraries(3rdparty_filament INTERFACE Threads::Threads ${CMAKE_DL_LIBS})
     if(UNIX AND NOT APPLE)
-        find_library(CPPABI_LIBRARY c++abi PATH_SUFFIXES
-            llvm-11/lib llvm-10/lib llvm-9/lib llvm-8/lib llvm-7/lib
-            REQUIRED)
-        get_filename_component(CPP_LIBDIR ${CPPABI_LIBRARY} DIRECTORY)
-        find_library(CPP_LIBRARY c++ PATHS ${CPP_LIBDIR} REQUIRED)
+        # Find CLANG_LIBDIR if it is not defined. Mutiple paths will be searched.
+        if (NOT CLANG_LIBDIR)
+            find_library(CPPABI_LIBRARY c++abi PATH_SUFFIXES
+                         llvm-11/lib llvm-10/lib llvm-9/lib llvm-8/lib llvm-7/lib
+                         REQUIRED)
+            get_filename_component(CLANG_LIBDIR ${CPPABI_LIBRARY} DIRECTORY)
+        endif()
+        # Find clang libraries at the exact path ${CLANG_LIBDIR}.
+        find_library(CPP_LIBRARY    c++    PATHS ${CLANG_LIBDIR} REQUIRED NO_DEFAULT_PATH)
```

Comment:
Getting this error in CI, though not sure why cmake did not stop here.
`-- CPP_LIBRARY: CPP_LIBRARY-NOTFOUND`

Reply:
this is strange, I added a check to see `CPP_LIBRARY-NOTFOUND` is true in CMake